src/components/converter/assets/dai.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
cc @bpierre Should we be using `tokenIconUrl()` from aragonUI? Or are we not using it at all in the microsite?

Reply:
Yes I think we could copy src/utils/web3.js for now?

Reply:
Ah yeah, let's do that!

Reply:
Sure!